packages/@aws-cdk/aws-eks/README.md

```diff
@@ -977,6 +977,18 @@ are being passed down (such as `repo`, `values`, `version`, `namespace`, `wait`,
 This means that if the chart is added to CDK with the same release name, it will try to update
 the chart in the cluster.
 
+Additionally, the `chartAsset` property can be an `aws-s3-assets.Asset`. This allows the use of local, private helm charts.
+
+```ts
+const chartAsset = new Asset(this, 'ChartAsset', {
+  path: path.join(__dirname, 'test-chart'),
+});
+this.cluster.addHelmChart('test-chart', {
+  chart: 'test-chart-asset',
```

Comment:
Is this really necessary? Seems like we can now make this an optional property, and validate that either `chart` or `chartAsset` is used.

Also - does `repository` and `version` have any meaning when using an asset? Should we assert they are not specified when using `chartAsset`?

Reply:
I agree with the point on making it an option property, then validating that either `chart` or `chartAsset`  is used.

For the second point, I don't believe so. I'll test that assertion out.

Reply:
I believe that `repository` should be `None`, but it would be another feature/issue to support versions for `chartAsset`. A possible solution would be to create nested file paths in s3, where some path references nested directories to scan for a specified version number.

Reply:
> I agree with the point on making it an optional property, then validating that either chart or chartAsset is used.

I don't see this change...am I missing something? We also need to validate the neither `repository` nor `version` is used if `chartAsset` is defined.